src/main/java/Event.java

```diff
@@ -1,13 +1,17 @@
+import java.util.Date;
+
 public class Event extends Task {
-    private String duration;
+    private Date start;
+    private Date end;
 
-    public Event(String description, String duration) {
+    public Event(String description, Date start, Date end) {
```

Comment:
I like how you improved on the Event type, by changing the description to include Start and End Date!